apps/pythonapi/examples/xarray_example.py

```diff
@@ -0,0 +1,61 @@
+# %% [md]
+#
+# # Rendering XArray data with Vapor
+#
+# Vapor supports render XArray data in a similar fashion to Numpy data
+# In order to pass XArray data to Vapor, create a data set of the type `vapor.dataset.PYTHON`
```

Comment:
The documentation says create a data set of type, vapor.dataset.PYTHON. But in the code below a convenience function, CreatePythonDataset(), is used to accomplish this task. The code and comment should be consistent.

Reply:
Fixed.